WebService/v1/Models/SimulationApiModel/SimulationStatistics.cs

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.IoTSolutions.DeviceSimulation.WebService.v1.Models.SimulationApiModel
+{
+    public class SimulationStatistics
+    {
+        [JsonProperty(PropertyName = "TotalMessagesSent")]
+        public long TotalMessagesSent { get; set; }
+
+        [JsonProperty(PropertyName = "AverageMessagesPerSecond")]
+        public double AverageMessagesPerSecond { get; set; }
```

Comment:
note, we've discussed this and the plan is to keep these stats for the last N minutes, so this value won't be a global average for the entire duration of the simulation. I'm wondering if we should make that explicit with a name like `CurrentAverageMessagesPerSecond` or similar - thoughts?

Reply:
These represent final state as ie when Simulation ends, we save TotalMessagesSent and AverageMessagesPerSecond in simulation def.